src/fromager/commands/step.py

```diff
@@ -109,9 +109,11 @@ def build_sdist(
 def _find_source_root_dir(
     work_dir: pathlib.Path,
     req: Requirement,
-    dist_version: str,
+    dist_version: Version,
 ) -> pathlib.Path:
-    source_root_dir = finders.find_source_dir(pathlib.Path(work_dir), req, dist_version)
+    source_root_dir = finders.find_source_dir(
+        pathlib.Path(work_dir), req, str(dist_version)
```

Comment:
nit pick: work_dir is already a pathlib.Path:

```suggestion
        work_dir, req, str(dist_version)
```